test-readme.md

```diff
@@ -1,13 +1,42 @@
-# Hello!
-My name is Ela Greenberg.
-This is my first exercise creating a repo.
-I will add more information here as time goes on. 
-
-In this file, I will learn markdown. I will learn it by:
-* Practicing
-* Trial and error
-* Copying examples and seeing their output.
-I will use [this reference](https://guides.github.com/features/mastering-markdown/) as a guide.
-
-One day **soon** I will be a markdown master.
+# Ela Greenberg
+
+_Jerusalem, Israel_
+
+[LinkedIn] (www.linkedin.com/in/elagreenberg)
+
+
+English language editor and aspiring technical communicator
+
+
+
+
+## Summary 
+* Over 10 years experience of editing and proofreading texts
+* Specializes in Middle East and Islamic studies, international relations policy fields, and non-profits
+* Works mostly with ESL authors
+* Native English speaker, fluent Hebrew, and basic Arabic
+
+## Experience
+**Dr. Ela Greenberg Editorial Services**
+_Copyeditor_
+* Engages in copyediting and proofreading
```

Comment:
change tense to imperative

Reply:
why? It doesn't make sense. I'm not telling somebody to copyedit.

Reply:
I changed it anyway, although I would not write a cv that way.